<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraGimbalControl</class>
 <widget class="QWidget" name="CameraGimbalControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>524</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>762</width>
    <height>524</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>12</number>
   </property>
   <item>
    <widget class="QGroupBox" name="cameraControl">
     <property name="title">
      <string>Camera Control</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <item>
       <widget class="QPushButton" name="photoButton">
        <property name="text">
         <string>Take Photo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="videoButton">
        <property name="text">
         <string>Start Video</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Gimbal Control</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QGroupBox" name="controlMode">
        <property name="title">
         <string>Control Mode</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QRadioButton" name="angleMode">
           <property name="text">
            <string>Angle</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">controlModeButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="speedMode">
           <property name="text">
            <string>Speed</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">controlModeButtonGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="angleControlMode">
        <property name="title">
         <string>Angle Control Mode</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QRadioButton" name="absoluteMode">
           <property name="text">
            <string>Absolute</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">angleControlButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="incrementatlMode">
           <property name="text">
            <string>Incremental</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">angleControlButtonGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QGroupBox" name="inputValues">
        <property name="title">
         <string>Input Values</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>12</number>
         </property>
         <property name="topMargin">
          <number>12</number>
         </property>
         <property name="rightMargin">
          <number>12</number>
         </property>
         <property name="bottomMargin">
          <number>12</number>
         </property>
         <item>
          <widget class="QFrame" name="rollControlValues">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="leftMargin">
             <number>6</number>
            </property>
            <property name="topMargin">
             <number>6</number>
            </property>
            <property name="rightMargin">
             <number>6</number>
            </property>
            <property name="bottomMargin">
             <number>6</number>
            </property>
            <property name="spacing">
             <number>6</number>
            </property>
            <item row="0" column="2">
             <widget class="QDoubleSpinBox" name="rollControlSpinBox"/>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="rollControlMin">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="rollControlModeText">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Roll (deg):</string>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="QLabel" name="rollControlMax">
              <property name="text">
               <string>100</string>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QSlider" name="rollControlSlider">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="rollCheckBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Enabled</string>
              </property>
             </widget>
            </item>
           </layout>
           <zorder>rollControlSlider</zorder>
           <zorder>rollControlModeText</zorder>
           <zorder>rollControlMin</zorder>
           <zorder>rollControlMax</zorder>
           <zorder>rollControlSpinBox</zorder>
           <zorder>rollCheckBox</zorder>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="pitchControlValues">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <property name="leftMargin">
             <number>6</number>
            </property>
            <property name="topMargin">
             <number>6</number>
            </property>
            <property name="rightMargin">
             <number>6</number>
            </property>
            <property name="bottomMargin">
             <number>6</number>
            </property>
            <property name="spacing">
             <number>6</number>
            </property>
            <item row="0" column="1">
             <widget class="QLineEdit" name="pitchControlModeText">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Pitch (deg) :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="QLabel" name="pitchControlMax">
              <property name="text">
               <string>100</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QDoubleSpinBox" name="pitchControlSpinBox"/>
            </item>
            <item row="0" column="4">
             <widget class="QSlider" name="pitchControlSlider">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="pitchControlMin">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="pitchCheckBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Enabled</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="yawControlValues">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <property name="leftMargin">
             <number>6</number>
            </property>
            <property name="topMargin">
             <number>6</number>
            </property>
            <property name="rightMargin">
             <number>6</number>
            </property>
            <property name="bottomMargin">
             <number>6</number>
            </property>
            <property name="spacing">
             <number>6</number>
            </property>
            <item row="0" column="5">
             <widget class="QLabel" name="yawControlMax">
              <property name="text">
               <string>100</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QDoubleSpinBox" name="yawControlSpinBox"/>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="yawControlModeText">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Yaw (deg) :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="yawControlMin">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QSlider" name="yawControlSlider">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="yawCheckBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Enabled</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QHBoxLayout" name="durationLayout">
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="durationLabel">
          <property name="minimumSize">
           <size>
            <width>319</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Time for gimbal to move to position: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="durationSpinBox">
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>sec</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sendGimbalCommand">
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="angleControlButtonGroup"/>
  <buttongroup name="controlModeButtonGroup"/>
 </buttongroups>
</ui>
